pkg/scheduler/scheduler.go

```diff
@@ -291,54 +291,52 @@ func (s *Scheduler) requeueClusterResourceBindings(selector labels.Selector) err
 
 func (s *Scheduler) getPlacement(resourceBinding *workv1alpha1.ResourceBinding) (policyv1alpha1.Placement, string, error) {
 	var placement policyv1alpha1.Placement
-	var clusterPolicyName string
-	var policyName string
-	var policyNamespace string
-	var err error
-	if clusterPolicyName = util.GetLabelValue(resourceBinding.Labels, policyv1alpha1.ClusterPropagationPolicyLabel); clusterPolicyName != "" {
-		var clusterPolicy *policyv1alpha1.ClusterPropagationPolicy
-		clusterPolicy, err = s.clusterPolicyLister.Get(clusterPolicyName)
-		if err != nil {
-			return placement, "", err
-		}
+	var policyName, clusterPolicyName string
 
-		placement = clusterPolicy.Spec.Placement
+	if policyName = util.GetLabelValue(resourceBinding.Labels, policyv1alpha1.PropagationPolicyNameLabel); policyName != "" {
```

Comment:
correct me if not. If `propagationPolicy` and `clusterPropagationPolicy` exist at the same time, we will prefer the `propagationPolicy`, so I move the logic forward.

Reply:
Why do you think that?

Reply:
Step1: we will get placement by `clusterPolicyName` from https://github.com/karmada-io/karmada/blob/master/pkg/scheduler/scheduler.go#L298-L306
Step2: if `policyName` exists, we will replace the former placement in https://github.com/karmada-io/karmada/blob/master/pkg/scheduler/scheduler.go#L308-L317
Step3: return the placement